src/components/dev-hub/filter-bar.js

```diff
@@ -70,7 +70,9 @@ export default ({ heading = 'All Articles' }) => {
             callStitch(metadata, 'products', setProducts);
         }
     }, [metadata, languages.length, products.length]);
-
+    const handleChange = (value, type) => {
+        setFilterValue({ ...filterValue, [type]: value });
```

Comment:
You might already have this in mind, but the filter changes should probably do something like `GatsbyRouter(or whatever their API is). routeTo('/langage/nodejs')`